low-level/files/opt/seagate/sspl/setup/conf_based_sensors_enable

```diff
@@ -73,14 +73,12 @@ if SSPL_STORE_TYPE == "consul":
         print("Error in connecting with consul: {}".format(cerror))
 
 # Update sensor information for sspl_test
-test_file_config_path=f"/opt/seagate/{PRODUCT_FAMILY}/sspl/sspl_test/conf/sspl_tests.conf"
-config = configparser.ConfigParser(allow_no_value=True)
-config.read(test_file_config_path)
+test_file_config_path=f"/opt/seagate/{PRODUCT_FAMILY}/sspl/sspl_test/conf/sspl_tests.yml"
+Conf.load('test', f'yaml://{test_file_config_path}')
```

Comment:
This may fail if any other component on this system already used this index.
Better keep the index as 'sspl_test'.

Reply:
Done.